### DISCUSSION TITLE
[Feature suggestion] Allow "First" and "Last" conflict handlers for 3.0.0

### DISCUSSION THREAD

**Discussion by DShip91** [Raytwo/ARCropolis#298] [Ideas]
### Have you read the disclaimer before proceeding?

- [X] I have read the disclaimer.

### What would the feature do?

This feature would allow users of 3.0.0 to prioritize files in either their romfs/arc folder or their ultimate/mods folder rather than being forced to disable entire mods because of one or two conflicting files.

### How should it be implemented?

Orbits already allows the "First" and "Last" conflict handlers by default so it should simply be a matter of adding the option somewhere in the config file.

**Comment by blu-dev** [comment]
I've considered adding this option, and if the user were to select those options then I would not be willing to assist with crashes because file conflicts *can* and *will* cause crashes occasionally, and if I see the config has that flag set I will not assist.

We are still discussing it, as for now it's going to stay the way it is.
